<!--igvc_run.launch-->
<launch>

<!-- Navigatin System CNS5000 -->
<node name="waypoint_provider" pkg="iggy_navigation" type="goal_pub_IGVC.py" output="screen">
	<remap from="navsat/fix" to="/cns5000/fix"/>   <!--Working -->
    <param name="waypoint_file" value="$(find iggy_navigation)/src/waypoints3.csv" />


</node>  

</launch>


